frontend/src/utils.js

```diff
@@ -10,3 +12,20 @@ export const postData = async (url, data) => {
 
   return response.ok;
 };
+
+export const useQuery = () => {
+  const queryString = useLocation().search;
+
+  return new URLSearchParams(queryString);
+};
+
+export const isAuthenticated = () => {
+  const token = window.localStorage.getItem("til_token");
```

Comment:
How about extracting "til_token" to a constant?

Reply:
Done.